scss/config/_abstracts.scss

```diff
@@ -2,7 +2,7 @@
 @use 'sass:meta';
 @use '../utilities';
 @use '~accoutrement/sass/tools' with (
-  $color-var-prefix: ''
```

Comment:
I don't really understand how this works, but is this still the right place for it?

Reply:
I think ideally we would have this set up like on OddSite and TimeDesigner where we are only loading a configured Accoutrement in one place and then `@use`ing that file in all the places we are currently loading Accoutrement. 

We also have this Accoutrement configuration pattern at the top of `config/index` in other projects (Estimating Engine, Top Dog). Would that be the best place to relocate this in the meantime @mirisuzanne ? Or should we re-work how we have Accoutrement set up before this is merged/released?

Reply:
@dvdherron I agree on both fronts:

- long term, we could update to match our usual pattern
- short term, we just need to make sure the configuration is the first use - so top of `config/index` makes sense for that

Reply:
fixed in https://github.com/oddbird/sassdoc-theme-herman/pull/390/commits/b57cff56dc07d0d93726344df8cf80e6afeb16a1